### PR TITLE
Sonarqube linkfix

### PR DESCRIPTION
**Pull request by shahariaazam**
Sonarqube linkfix

### CODE REVIEW

**Comment by tianon** [comment]
cc `sonarqube` image maintainers for verification:
@janos-ss
@julienlancelot
@christophelevis
@pierre-guillot-sonarsource
@michal-duda-sonarsource

File: sonarqube/content.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ $ mvn sonar:sonar
 $ mvn sonar:sonar -Dsonar.host.url=http://$(boot2docker ip):9000
 ```
 
-To analyze other kinds of projects and for more details see [Analyzing Source Code documentation](https://redirect.sonarsource.com/doc/analyzing-source-code.html).
+To analyze other kinds of projects and for more details see [Analyzing Source Code documentation](https://docs.sonarqube.org/latest/analysis/overview/).
```
Comment (julienlancelot) [inline sonarqube/content.md]: We prefer using "redirect.sonarsource.com" as often os possible in order to more easily update links when needed.
So please keep usage of "redirect.sonarsource.com"

File: sonarqube/content.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ By default, the image will use an embedded H2 database that is not suited for pr
 
 > Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
 
-Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
+Setup a database by following the [Installing the Database](https://docs.sonarqube.org/latest/setup/install-server/) section.
```
Comment (julienlancelot) [inline sonarqube/content.md]: 👍

File: sonarqube/content.md (hunk 1 above)
Comment (shahariaazam) [reply sonarqube/content.md]: Changes have been made.

**Pull request merged**